src/server/auth/server/testing/auth_test.go

```diff
@@ -1238,8 +1241,9 @@ func TestListRepoNotLoggedInError(t *testing.T) {
 		buildBindings(alice, auth.RepoOwnerRole), getRepoRoleBinding(t, aliceClient, repo))
 
 	// Anon (non-logged-in user) calls ListRepo, and must receive an error
-	_, err := anonClient.PfsAPIClient.ListRepo(anonClient.Ctx(),
+	c, _ := anonClient.PfsAPIClient.ListRepo(anonClient.Ctx(),
```

Comment:
No error check?

Reply:
I guess the first error should always be nil.